vita-latina.csl

```diff
@@ -0,0 +1,305 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="fr-FR">
+  <!--
+
+Feuille de style CSL 
+à destination du logiciel Zotero 
+comportant les instructions de formatage des références bibliographiques et de la bibliographie de littérature secondaire définies par la revue Vita Latina [http://www.persee.fr/collection/vita ; http://vitalatina.upv.univ-montp3.fr/]
+réalisée par Louis Autin (doctorant - Université Grenoble Alpes ; Universität Osnabrück)
+
+Remarques :
+	- pour les thèses, Zotero ne permet pas de spécifier le directeur ; la feuille de style laisse donc cette indication à compléter manuellement, entre crochets ;
+	- à ce stade, le langage CSL ne permet pas de définir certains éléments de formatage, tels que la police, la taille de la police ou l'alignement de texte (ainsi, forcer la bibliographie à être justifiée). Ces éléments doivent être mis en place manuellement. 
+ -->
+  <!-- INFORMATIONS RELATIVES A LA FEUILLE DE STYLE -->
+  <info>
+    <title>Vita Latina (French)</title>
+    <title-short>VL</title-short>
+    <id>http://www.zotero.org/styles/vita-latina</id>
+    <link href="http://www.zotero.org/styles/vita-latina" rel="self"/>
```

Comment:
@Oxil-2, before we accept this, could you add a "documentation" link that points to instructions to authors? (probably to http://vitalatina.upv.univ-montp3.fr/page-3/, I think). See https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link

And, if you based this style on an existing CSL style from the repository, please also add a "template" link. See https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link

Finally, can you add the electronic ISSN as well? Looks like that's "1783-1830". See https://github.com/citation-style-language/styles/wiki/Style-Requirements#9---issn-and-eissn